tap_mixpanel/__init__.py

```diff
@@ -47,7 +47,14 @@ def main():
         LOGGER.warning(
             "WARNING: Setting start_date to 1 year ago, {}".format(start_date))
 
+    #Check support for EU endpoints
+    if parsed_args.config.get('eu_residency_server'):
```

Comment:
@prijendev Please update this condition as per https://github.com/singer-io/tap-xero/blob/master/tap_xero/streams.py#L122

Reply:
done